modules/EnsEMBL/Web/Component/Help/Contact.pm

```diff
@@ -90,7 +90,7 @@ sub content {
     'label'     => 'What is 5 + 4? ',
     'value'     => uri_unescape($hub->param('logic')) || '',
     'required'  => 1,
-    'helptip'     => 'Are you a human verification'
+    'helptip'     => 'Question to verify you are a human or bot'
```

Comment:
`Question to verify you are a human or bot` -> `Question to verify that you are a human` (user probably shouldn't read that we may consider them a bot)